src/main/scala/no/ndla/frontpageapi/model/api/UpdatedAboutSubject.scala

```diff
@@ -0,0 +1,13 @@
+package no.ndla.frontpageapi.model.api
+
+import no.ndla.frontpageapi.model.domain.LanguageField
+
+//TODO er det dumt om languagefield extendes på noe som ikke alle er languagefields? kan jeg endre så det bare er description som er en languagefield?
+case class UpdatedAboutSubject(title: String,
```

Comment:
Trur det korrekte her blir å lage en `FrontpageTitle(title: String, language: String) extends LanguageField` og en tilsvarende FrontpageDescription.

Reply:
Men da må jeg gjøre tilsvarende med NewAboutSubject? Eller er det greit at de forskjellige modellene er litt ulike på akkurat det, siden det uansett er ulike converter-metoder?

Reply:
FilmFrontPage bruker også domain-modellen AboutSubject, og det funker nå med multiple languages når hele AboutSubject extender LanguageField, så da er det kanskje like greit å la det stå?

Reply:
Men FilmFrontPage bruker også domain-modellen AboutSubject, og det funker nå med multiple languages når hele AboutSubject extender LanguageField. Kan lage en egen AboutSubject for SubjectFrontpage eventuelt?